### DISCUSSION TITLE
Question about transform hover effect

### DISCUSSION THREAD

**Discussion by Chandler-Zhu** [tailwindlabs/tailwindcss#5069] [Help] [answered]
I made a skew card section, now I want to add a hover effect which remove the skew for the card and the text when user hover on it, but I run into a problem, when I hover on the padding area, the card looks fine but the text is still skewed. How should I fix this ?
Here is my code

https://jsfiddle.net/sn2km473/

**Comment by dance2die** [comment, marked as answer]
You can [group hover effects](https://tailwindcss.com/docs/hover-focus-and-other-states#group-hover). 
**demo**: https://play.tailwindcss.com/Z1jzTmxfg7

I used Tailwind Play instead of jsfiddle, because `group-hover` isn't available in the linked tailwind.min.css(It needs JIT mode or need to have "skew" variant to be specified during run/build/dev time).

---

@Chandler-Zhu: Updated the play link! Sorry. I forgot to use the hardlink.
